docs/en/core-services/Caching.md

```diff
@@ -321,8 +321,8 @@ To enforce the assumption that tiers are configured from fastest to slowest, All
 blocks around tiers based on block annotation policies.
 
 Below configuration is honoured by each individual tier management task:
-- `alluxio.worker.management.task.thread.count`: How many threads to use for management tasks. (Default: `CPU core count`)
-- `alluxio.worker.management.block.transfer.concurrency.limit`: How many block transfers can execute concurrently. (Default:`CPU core count`/ 2)
+- `alluxio.worker.management.task.thread.count`: How many threads to use for management tasks. (Default: the maximum value of `4` and `CPU core count`)
+- `alluxio.worker.management.block.transfer.concurrency.limit`: How many block transfers can execute concurrently. (Default: the maximum value of  `2` and `CPU core count / 2`)
```

Comment:
```suggestion
- `alluxio.worker.management.block.transfer.concurrency.limit`: How many block transfers can execute concurrently. (Default: the maximum value of `2` and `CPU core count / 2`)
```

Reply:
i wonder if it is better to reference the properties list doc as the source of truth for the default value: https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html

otherwise this part will be out of date until someone manually catches it.

Reply:
@Xenorith is it possible to put an html anchor to each property? otherwise it might be a little bit difficult for the users to navigate to the right place of the property.

Reply:
Or is there a better approach to embed a property snippet with its default value and comments in the doc?  I just found there are quite a few properties in this doc, is it awkward if I put a link for each of them?

Reply:
i think it is possible with some jekyll magic

Reply:
i don't know about other embedding approaches though. but providing an anchor link seems useful in general; one less than step compared to going to page then searching the entire page by copying the property key

Reply:
oh i just tried to manually add in the anchor and it works already! looks like it was added a while back in https://github.com/Alluxio/alluxio/pull/6582

ex. https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html#alluxio.worker.management.block.transfer.concurrency.limit

Reply:
Awesome, I will update this pr to add an anchor links, thx!

Reply:
@Xenorith the link added